indium-webkit.el

```diff
@@ -235,11 +236,17 @@ Allowed states: `\"none\"', `\"uncaught\"', `\"all\"'."
   (indium-webkit--send-request `((method . "Network.setCacheDisabled")
                                  (params . ((cacheDisabled . ,(if disabled t :json-false)))))))
 
-(defun indium-webkit-set-cache-disabled (arg)
-  "Disable network cache for each request.  If prefix ARG is used enable it."
-  (interactive "p")
-  (setq indium-webkit-cache-disabled (= arg 1))
-  (indium-webkit--set-cache-disabled indium-webkit-cache-disabled))
+(defun indium-webkit-enable-cache ()
+  "Enabled network cache for each request.  And save it for future sessions."
```

Comment:
I'm being picky here, but I'd like the second sentence to start on a separate line, and not starting with "And".
Also, what does "future sessions" mean? It's not persistent if the user restarts Emacs for instance.